scripts/flail.py

```diff
@@ -136,6 +155,25 @@ def run(self, output_file, header = None, macro : str = None):
 
 
 def parse_args(argv):
+    '''
+    Try the following examples:
+
+    Ex.1 Convert a list of C operational models (see last line of the CMD below) into unsigned int arrays.
+         Write the resulting array into libc.c and libc.h
+         CMD:
+            python3 flail.py \
+                --macro ESBMC_FLAIL \
+                --prefix esbmc_libc_ \
+                -o ./libc.c \
+                --header ./libc.h \
+                esbmc/src/c2goto/library/builtin_libs.c src/c2goto/library/ctype.c
+
+    Ex.2 Convert a goto binary file (see last line of the CMD below) into an unsigned int arrays
+         After building ESBMC, find clib32.goto from the build tree
+         CMD:
+            python3 flail.py -o ./clib32.c \
+                    esbmc/build/src/c2goto/clibd32.goto
+    '''
```

Comment:
I suggest either move this as a doc for the module itself or add it as as `help` options

Reply:
Done